src/main/java/com/xiaomi/infra/pegasus/rpc/async/TableHandler.java

```diff
@@ -389,6 +389,14 @@ public void asyncOperate(client_operator op, ClientOPCallback callback, int time
 
     ClientRequestRound round =
         new ClientRequestRound(op, callback, manager_.counterEnabled(), (long) timeoutMs);
+
+    if (op.enableSizeLimit
+        && manager_.getMaxAllowedWriteSize() > 0
+        && round.operator.header.body_length > manager_.getMaxAllowedWriteSize()) {
+      round.operator.rpc_error.errno = error_types.ERR_INVALID_DATA;
```

Comment:
What's `ERR_INVALID_DATA` mean? Do it same with the error code that rdsn will return when the body size is over allowed write size?

Reply:
Yeah, it same to rdsn.